src/Pages/naeunsong/Login/Login.scss

```
@@ -0,0 +1,65 @@
+@import url('https://fonts.googleapis.com/css2?family=Lobster&display=swap');
+
+* {
+  box-sizing: border-box;
+}
+
+a {
+  text-decoration: none;
+}
+
+input, button{
+  outline: none;
+}
+
+.login{
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+  width: 350px;
+  height: 380px;
+  margin: auto;
+  border: 1px solid #dbdbdb;
+  background-color: white;
+  text-align: center;
+  
+  .logo{
+    background-color: white;
+    font-family: 'Lobster', cursive;
+    font-size: 40px;
+  }
+    
+  input{
+    width: 70%;
+    margin: 4px;
+    padding: 9px 0 7px 8px;
+    background-color: rgba(250,250,250,1);
+    border: 1px solid #dbdbdb;
+    border-radius: 4px;
+    line-height: 18px;
+    font-size: 12px;
+  }
+    
+  button {
+    width: 70%;
+    margin: 8px 0px 60px 0px;
```

아래와 같이 작성해주실 수도 있습니다.
```suggestion
    margin: 8px 0px 60px;
```

---

오호 세 개도 되는지는 몰랐네요! 감사합니다!!